src/AbstractCollection.php

```diff
@@ -59,14 +57,4 @@ public function isEmpty(): bool
     {
         return empty($this->_loaded);
     }
-
-    /**
-     * Returns iterator.
-     *
-     * @return \ArrayIterator
-     */
-    public function getIterator(): ArrayIterator
```

Comment:
How does one iterate these collections now?

Reply:
This collection extends core's `ObjectRegistry` which already has it.